web/profiles/joinup/src/Form/UserCancelForm.php

```diff
@@ -61,45 +60,54 @@ public static function create(ContainerInterface $container) {
    * {@inheritdoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
```

Comment:
As we're changing a lot here, let's move the whole class in `joinup_group` where this really belongs. Also the corresponding part from `joinup_entity_type_alter()` should follow there.